drop-in/src/main/java/com/adyen/checkout/dropin/ui/action/ActionComponentDialogFragment.kt

```diff
@@ -91,12 +92,8 @@ class ActionComponentDialogFragment : DropInBottomSheetDialogFragment(), Observe
                 }
             }
 
-            if (!isHandled) {
-                (actionComponent as ActionComponent<*>).handleAction(requireActivity(), action)
-                isHandled = true
-            } else {
-                Logger.d(TAG, "action already handled")
-            }
+            actionComponent.handleAction(requireActivity(), action)
+            attachComponent(actionComponent, binding.componentView)
```

Comment:
Wouldn't it be better to turn these 2 lines around? That way the view doesn't miss out on anything that happens in `handleAction`

Reply:
It doesn't work with the generic component, because until we call `handleAction` the delegate is not created yet, and `attachComponent` requires that the delegate is already created. Even creating an `ActionDelegate` might not solve this :/